bip-update-process.md

```diff
@@ -0,0 +1,705 @@
+```
+BIP: ?
+Title: Updated BIP Process
+Author: Murch <murch@murch.one>
+Comments-URI: https://github.com/bitcoin/bips/wiki/Comments:BIP-Updated-BIP-Process
+Status: Draft
+Type: Process
+Created: 2024-05-13
+License: BSD-2-Clause
+Post-History: https://github.com/murchandamus/bips/pull/2
+Requires: 123
+Replaces: 2
+```
+
+## Abstract
+
+This _Bitcoin Improvement Proposal (BIP)_ provides information about the preparation of BIPs and policies relating to
+the publication of BIPs. It replaces [BIP 2](bip-0002.mediawiki) with a streamlined process, and may be amended to
+address the evolving needs of the BIP process.
+
+## Motivation
+
+BIP 2 is over eight years old and was written when different concerns were pressing to the bitcoin[^capitalization]
+community. The BIP process as defined by BIP 2 thus far seems to have been fashioned to facilitate the design and
+activation of protocol changes. In the past years, BIPs more often describe interoperable features beyond the base
+protocol. The community has had multiple debates about the role of BIP Editors, and some aspects of the process
+specified by BIP 2 that did not seem to achieve the intended goals. This proposal sunsets aspects of the BIP 2 process
+that did not achieve broad adoption, reduces the judgment calls assigned to the BIP Editor role, and delineates the
+BIP Types more expediently.
+
+## Fundamentals
+
+### What is a BIP?
+
+BIPs cover the range of interests of the bitcoin community. The main topic is technology that supports the bitcoin
+currency. Most BIPs provide a concise, self-contained, technical description of one new concept, feature, or standard.
+Some BIPs describe processes, implementation guidelines, best practices, incident reports (e.g.
+[BIP 50](bip-0050.mediawiki)), or other information relevant to the bitcoin community. However, any topics related to
+the bitcoin protocol, bitcoin’s peer-to-peer network, and bitcoin client software may be acceptable.
+
+BIPs are intended to be the primary mechanism for proposing new protocol features, coordinating client standards, and
+documenting design decisions that have gone into implementations. BIPs may be submitted by anyone.
+
+### BIP Ownership
+
+Each BIP is primarily owned by its authors and represents the authors’ opinion or recommendation. The authors are
+expected to foster discussion, address feedback and dissenting opinions, and, if applicable, advance the adoption of
```

Comment:
To what degree do we want the BIP PR request to be the place where discussion about a BIP happens vs the mailinglist? Is it appropriate to suggest alternative designs in the PR comments?

Reply:
I am not sure that there is a clear line here. I’d say conceptual review and approach discussion should happen preferably in response to the initial presentation of the new idea, but it may still be on-topic in the pull request. If it requires a complete overhaul, it may be better to bring it to the mailing list, as an alternate design that requires a complete rewrite might not be actionable in the context of a pull request.

Do you think there is something missing here that should be clarified? Do you have a concrete suggestion?

Reply:
You answered my question. I wasn't sure if this BIP was attempting provide a concrete position of type of feedback. That had been the case I would've recommend stating what is allowed more explicitly. However given that it is isn't the case, it is reasonable as written.